src/components/Match/matchColumns.jsx

```diff
@@ -706,3 +660,132 @@ export const teamfightColumns = [
     displayFn: inflictorRow(items),
   },
 ];
+
+export const visionColumns = [
+  heroTdColumn, {
+    center: true,
+    displayName: (
+      <div style={{ display: 'inline-flex', verticalAlign: 'middle' }}>
+        <img height="15" src={`${API_HOST}/apps/dota2/images/items/ward_observer_lg.png`} role="presentation" />
+        &nbsp;{strings.th_purchase_shorthand}
+      </div>
+    ),
+    tooltip: strings.tooltip_purchase_ward_observer,
+    field: 'purchase_ward_observer',
+    sortFn: true,
+    displayFn: (row, col, field) => field || '-',
+  }, {
+    center: true,
+    displayName: (
+      <div style={{ display: 'inline-flex', verticalAlign: 'middle' }}>
+        <img height="15" src={`${API_HOST}/apps/dota2/images/items/ward_observer_lg.png`} role="presentation" />
+        &nbsp;{strings.th_use_shorthand}
+      </div>
+    ),
+    tooltip: strings.tooltip_used_ward_observer,
+    sortFn: true,
+    displayFn: row => row.item_uses.ward_observer || '-',
```

Comment:
this will crash if item_uses is undefined (unparsed match).  The vision tab won't be clickable if this is the case, but may want to handle it for safety/if the user direct links to it.